src/server/kernel/wxwaylandsurface.cpp

```diff
@@ -392,6 +395,27 @@ QRect WXWaylandSurface::getContentGeometry() const
     return QRect(0, 0, d->nativeHandle()->width, d->nativeHandle()->height);
 }
 
+QString WXWaylandSurface::title() const
+{
+    W_DC(WXWaylandSurface);
+
+    return { d->nativeHandle()->title };
```

Comment:
QString::fromUtf8 ?

What's the text encoding of title?

Reply:
copy from xdgsurface @justforlxz

Reply:
It must be  utf8